examples/phi3/README_VISION.md

```diff
@@ -0,0 +1,50 @@
+# Phi3 optimization with Olive
+This folder contains an example of optimizing the microsoft/Phi-3-vision-128k-instruct model from [Hugging Face](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct) with Olive.
+
+## Prerequisites
+Please refer to the [pre-requisites of Phi3-vision optimization with GenAI](https://github.com/microsoft/onnxruntime-genai/blob/main/examples/python/phi-3-vision.md#0-pre-requisites) for more details.
+Basically, you need to install the following packages:
+- `huggingface_hub[cli]`
```

Comment:
maybe the requirements other than ort-nightly can be moved into a requirement.txt file? Otherwise, the user needs to copy paste each one

Reply:
Updated.